DAT_files/hd183143.dat

```diff
@@ -13,7 +13,8 @@ WISE1 = 3.307 +/- 0.478 ; WISE ref = IRSA AllWISE Source Catalog
 WISE2 = 2.817 +/- 0.483
 WISE3 = 3.053 +/- 0.012
 WISE4 = 2.768 +/- 0.021
-sptype = B7Iae ; sptype ref = Hobbs+2009 (2009ApJ...705...32H)
+sptype = B7Ia ; sptype ref = Morgan, Code & Whitford 1955 (1955ApJS....2...41M)
 corfac_spex_SXD = 0.593
 LXD_man = True
 corfac_spex_LXD = 0.62
+#spex_mask = [(1.347,1.411),(1.801,1.942),(2.546,2.86),(4.176,4.583)]
```

Comment:
Commented out.  Should it be included then?

Reply:
Nope, this was a leftover from a test. I removed it now.

Reply:
There other files with `#spex_mask` lines.  Maybe you could check those as well?  I only commented on the first.

Reply:
Oops, removed those as well!